server/src/main/java/io/crate/planner/statement/CopyFromPlan.java

```diff
@@ -85,7 +86,7 @@
 import static io.crate.analyze.CopyStatementSettings.settingAsEnum;
 import static io.crate.analyze.GenericPropertiesConverter.genericPropertiesToSettings;
 
-public final class CopyFromPlan implements Plan {
+public final class CopyFromPlan extends CopyPlan implements Plan {
```

Comment:
No need to make `CopyPlan` abstract and use inheritance here to re-use a static method.